src/main/java/edu/kis/powp/command/ComplexCommand.java

```diff
@@ -0,0 +1,25 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.ArrayList;
+
+public class ComplexCommand implements DriverCommand {
+    ArrayList<DriverCommand> driverCommands;
```

Comment:
Should be private!
List type (attribute and in the constructor) would be sufficient and would not limit the implementation.